examples/doc_examples/quickstart/simple_counter_testcases.py

```diff
@@ -0,0 +1,180 @@
+"""
+This file is public domain, it can be freely copied without restrictions.
+SPDX-License-Identifier: CC0-1.0
+"""
+
+# Imports for all Quickstart examples
+import cocotb
+from cocotb import start_soon
+from cocotb.clock import Clock
+from cocotb.triggers import FallingEdge, NextTimeStep, ReadOnly, RisingEdge, Timer
+
+# QUICKSTART 1
+
+
+@cocotb.test()
+async def quickstart_1(dut):
+    """
```

Comment:
The docstrings should follow https://peps.python.org/pep-0257/#multi-line-docstrings